osu.Game/OsuGame.cs

```diff
@@ -501,6 +503,23 @@ private void load()
         /// <param name="version">The build version of the update stream</param>
         public void ShowChangelogBuild(string updateStream, string version) => waitForReady(() => changelogOverlay, _ => changelogOverlay.ShowBuild(updateStream, version));
 
+        /// <summary>
+        /// Present ether a skin select if one skin was imported or navigate to the skin selection menu if multiple skins were imported immediately.
+        /// </summary>
+        /// <param name="importedSkins">List of imported skins</param>
+        public void PresentSkins(IEnumerable<Live<SkinInfo>> importedSkins)
+        {
+            int importedSkinsCount = importedSkins.Count();
+
+            if (importedSkinsCount == 1)
+                SkinManager.CurrentSkinInfo.Value = importedSkins.Single();
+            else if (importedSkinsCount > 1)
+            {
+                Settings.Show();
+                Settings.SectionsContainer.ScrollTo(Settings.SectionsContainer.Single(container => container is SkinSection));
```

Comment:
A nice follow up would be to highlight added skins in the dropdown, or add a marker to show that they are new, or move them to the top above the default alphabetical ordering. Probably requires further UX discussion so not happening in this PR.

Reply:
I'm also wondering if having this how you have it is wise. It doesn't match the implementation of beatmap and score presentation which also select the first item. That would arguably be better behaviour.

Also you are doing linq logic here which has the potential to silently fail (there's no test coverage of what you've added, so say `SkinSection` goes missing, it will cause a crash or error to the end user).

I'd propose removing this for now and using the same logic at `PresentBeatmap` / `PresentScore`.

Reply:
> there's no test coverage of what you've added
I would like to add testing to this, but there is no tools for this. `Skinmanager` is not even implemented into `TestOsuGame`. Ofc ' i can try to add required functionality by my own, but due to lack of Osu! codebase acknowledge it would take a while :/

Reply:
For now could you please follow the proposal above of having it always select the first skin? We can iterate on that as a future change.

> I would like to add testing for this, but there is no tools to do so.

Check out the "Navigation" category of tests, which actually test full game scenarios (and use the full `OsuGame` to do so, providing much of the functionality, including what you likely want). But I don't think it's required for this PR if you apply the above changes.